src/lang/modifyAst/deleteFromSelection.ts

```diff
@@ -170,6 +171,7 @@ export async function deleteFromSelection(
       selection.artifact &&
       selection.artifact.type !== 'sweep' &&
       selection.artifact.type !== 'plane' &&
+      selection.artifact.type !== 'compositeSolid' &&
```

Comment:
@Irev-Dev I just added to the existing codepath here for point and click var declarations 🤷